### PR TITLE
Clarify instructions for running the CLI from docker

### PR DESCRIPTION
**Pull request by kdhamric**
Clarified the instructions on how to run the CLI from docker, including explaining that it would need to connect to a remote Tracetest server.

### CODE REVIEW

File: docs/docs/ci-cd-automation/overview.md [hunk 1]
```diff
@@ -14,12 +14,18 @@ If you want to see more examples with other CI/CD tools, let us know by [opening
 
 Tracetest is designed to work with all CI/CD platforms and automation tools. To enable Tracetest to run in CI/CD environments, make sure to [install the Tracetest CLI](../getting-started/installation.mdx) and configure it to access your [Tracetest server](../configuration/server.md).
 
-You can also directly execute the Tracetest CLI from a Docker image rather than installing the CLI on your local machine. This can be convenient when you wish to execute the CLI in a CI/CD environment.
+### Running Tracetest CLI From Docker
```
Comment (jfermi) [inline docs/docs/ci-cd-automation/overview.md:17]:
```suggestion
### Running Tracetest CLI from Docker
```

File: docs/docs/ci-cd-automation/overview.md [hunk 2]
```diff
@@ -14,12 +14,18 @@ If you want to see more examples with other CI/CD tools, let us know by [opening
...
+
+Many integrations with CI/CD tools can be accomplished by running the [Tracetest CLI](../cli/configuring-your-cli) to execute a test against a remote Tracetest server. If you do not want to install the Tracetest CLI in your environment, you can choose to directly execute it from a Docker image. 
 
 **How to Use**:
 
+Use the command below, substituting the following placeholders:
+- <your-tracetest-server-url> - the url to the running Tracetest server you wish to execute the test on. Example: http://localhost:11633/
```
Comment (jfermi) [inline docs/docs/ci-cd-automation/overview.md:24]:
```suggestion
- <your-tracetest-server-url> - the URL to the running Tracetest server you wish to execute the test on. Example: http://localhost:11633/
```

File: docs/docs/ci-cd-automation/overview.md [hunk 3]
```diff
@@ -14,12 +14,18 @@ If you want to see more examples with other CI/CD tools, let us know by [opening
...
+- <file-path> - the path to the saved Tracetest test. Example: ./mytest.yaml
```
Comment (jfermi) [inline docs/docs/ci-cd-automation/overview.md:25]:
```suggestion
- <file-path> - The path to the saved Tracetest test. Example: ./mytest.yaml
```

**Review by mathnogueira — APPROVED** [review]

File: docs/docs/cli/configuring-your-cli.md [hunk 4]
```diff
@@ -65,8 +65,12 @@ There are times when it is easier to directly execute the Tracetest CLI from a D
 
 **How to Use**:
 
+Use the command below, substituting the following placeholders:
+- <your-tracetest-server-url> - the url to the running Tracetest server you wish to execute the test on. Example: http://localhost:11633/
```
Comment (jfermi) [inline docs/docs/cli/configuring-your-cli.md:69]:
```suggestion
- <your-tracetest-server-url> - The URL to the running Tracetest server you wish to execute the test on. Example: http://localhost:11633/

```

File: docs/docs/cli/configuring-your-cli.md [hunk 5]
```diff
@@ -65,8 +65,12 @@ There are times when it is easier to directly execute the Tracetest CLI from a D
 
 **How to Use**:
 
+Use the command below, substituting the following placeholders:
+- <your-tracetest-server-url> - the url to the running Tracetest server you wish to execute the test on. Example: http://localhost:11633/
+- <file-path> - the path to the saved Tracetest test. Example: ./mytest.yaml
```
Comment (jfermi) [inline docs/docs/cli/configuring-your-cli.md:70]:
```suggestion
- <file-path> - The path to the saved Tracetest test. Example: ./mytest.yaml

```

**Review by jfermi — COMMENTED** [review]
A few editing suggestions.

**Review by xoscar — APPROVED** [review]

**Pull request merged**